patterns/hero-product-split.php

```diff
@@ -0,0 +1,32 @@
+<?php
+/**
+ * Title: WooCommerce Hero Product - Split
+ * Slug: woocommerce-blocks/hero-product-split
+ * Categories: WooCommerce
+ */
+?>
+
+<!-- wp:group {"layout":{"type":"constrained"}} -->
+<div class="wp-block-group">
+	<!-- wp:media-text {"align":"full","mediaPosition":"right","mediaType":"image","mediaSizeSlug":"full","imageFill":false,"backgroundColor":"contrast","textColor":"base"} -->
+	<div class="wp-block-media-text alignfull has-media-on-the-right is-stacked-on-mobile has-base-color has-contrast-background-color has-text-color has-background">
+		<div class="wp-block-media-text__content"><!-- wp:heading -->
+			<h2 class="wp-block-heading">Get cozy this fall with<br>flannel shirts</h2>
+			<!-- /wp:heading -->
+
+			<!-- wp:buttons {"style":{"spacing":{"margin":{"bottom":"var:preset|spacing|40"}}}} -->
+			<div class="wp-block-buttons" style="margin-bottom:var(--wp--preset--spacing--40)">
+				<!-- wp:button -->
+				<div class="wp-block-button"><a class="wp-block-button__link wp-element-button">Shop now</a></div>
+				<!-- /wp:button -->
+			</div>
+			<!-- /wp:buttons -->
+		</div>
+
+		<figure class="wp-block-media-text__media">
+			<img src="https://images.unsplash.com/photo-1610873397492-e29c42344c00?ixlib=rb-4.0.3&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1277&q=80" alt="Woman in red, black, and white plaid hoodie." />
```

Comment:
I'm not sure which is the right approach here, just wondering if we want to link to external images or include them on the plugin like the approach I took here https://github.com/woocommerce/woocommerce-blocks/pull/9180/files ? 🤔

Reply:
Thanks! Updated in c9d9e71.